CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ target_include_directories(p4est PUBLIC
   $<INSTALL_INTERFACE:include>)
 target_link_libraries(p4est PUBLIC SC::SC )
 
-if (P4EST_HAVE_WINSOCK2_H)
+if ( WIN32)
```

Comment:
Thanks, happy to merge! Is this supposed to be a space (in this case please remove) or an underscore?

Reply:
The latest commit removes the space, looking forward to the merge :)